temp_examples/scenic-streetmap-overlay/scenic_plot.html

```diff
@@ -0,0 +1,398 @@
+            mouseover: highlightFeature,
+            mouseout: resetHighlight,
+            click: displayInfo
+          });
+        }
+
+	$.getJSON("lsoa_boundary_and_crime_data_30_10_19.geojson", function(json){
+          lsoaBoundaries = L.geoJson(json, { // boundary data as well as income, health data etc. info to display
+            style: function (feature) {
+              return {
+                fillColor: '#acadc1',
+                weight: 1,
+                opacity: 0,
+                color: 'white',
+                fillOpacity: 0
+              };
+            },
+            onEachFeature: onEachFeature
+          }).addTo(leafletMap);
+
+          var sceneryLayer = L.geoJson(json, { // scenery layer displaying mean scenic ratings per LSOA
+            style: function (feature) {
+              return {
+                fillColor: percToColour(feature.properties.scenic_rating, 2.16, 4.84),
+                weight: 1,
+                opacity: 0,
+                color: 'white',
+                dashArray: '1',
+                fillOpacity: .6
+              }; 
+            }
+          });
+          
+          var crimeLayer = L.geoJson(json, { // crime layer displaying mean monthly crime counts per LSOA
+            style: function (feature) {
+              return {
+                fillColor: getCrimeColour(feature.properties.mean_monthly_crime_count),
+                weight: 1,
+                opacity: 0,
+                color: 'white',
+                dashArray: '1',
+                fillOpacity: .4
+              }; 
+            }
+          });
+
+          var d3Layer = L.Class.extend({ // extend leaflet class to toggle on/off d3 layer w/ other leaflet layers
+            initialize: function() {
+              return;
+            },
+            onAdd: function() {
+              leafletMap.on('viewreset', mapmove); // remove points and redraw relevant subset as we move around map
+              redrawSubset(geoData.features); // draw initial susbet in starting position
+            },
+            onRemove: function() {
+              leafletMap.off('viewreset', mapmove);
+              d3.selectAll(".point").remove();
+            },
+          });
+          svgPointsLayer = new d3Layer(); 
+          
+          // switch between scenic rating per points vs mean per LSOA vs crime counts
+          var baseMaps = {
+            "Average Scenic Ratings per LSOA": sceneryLayer,
+            "Average Crime Count per Month": crimeLayer,
+            "Scenic Points": svgPointsLayer
+          };
+          L.control.layers(baseMaps, null, {position: 'bottomleft', collapsed: false}).addTo(leafletMap);
+          leafletMap.attributionControl.addAttribution('Scenic Data &copy; <a href="http://scenicornot.datasciencelab.co.uk/">Scenic-Or-Not</a>');
+        });
+
+        /**
+         * Then, add and scale info bars (scenic rating and indices of health, income and employment deprivation) 
```

Comment:
It would be great to include data from the other deprivation variables e.g. education. I think we restricted this before only because we were connecting the data to sound(!) so just wanted to keep it simple.